src/Monitoring/grafana-init/grafana.ini

```diff
@@ -0,0 +1,62 @@
+
+#################################### Server ####################################
+[server]
+# Protocol (http, https, h2, socket)
+protocol = http
+
+# The http port  to use
+http_port = 3000
+
+# The public facing domain name used to access grafana from a browser
+domain = dotnet-eng-grafana.westus2.cloudapp.azure.com
+
+# Redirect to correct domain if host header does not match domain
+# Prevents DNS rebinding attacks
+enforce_domain = true
+
+# The full public facing url you use in browser, used for redirects and emails
+# If you use reverse proxy and sub path specify full url (with sub path)
+root_url = https://%(domain)s/
+
+#################################### Users ###############################
+[auth]
+# Set to true to disable (hide) the login form, useful if you use OAuth, defaults to false
+disable_login_form = true
+
+# Set to true to attempt login with OAuth automatically, skipping the login screen.
+# This setting is ignored if multiple OAuth providers are configured.
+;oauth_auto_login = false
+
+#################################### Generic OAuth ##########################
+[auth.generic_oauth]
+enabled = true
+name = Azure AD
+allow_sign_up = true
+client_id = a2541735-8225-40af-9c8a-2ae233203739
+;client_secret = <managed by environment>
+scopes = openid email name
+auth_url = https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db$
+token_url = https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011d$
```

Comment:
Are these URLs right? They seem truncated.

Reply:
They are indeed. Copy/pasting through putty is a mess. But I think I fixed them.